### DISCUSSION TITLE
What is the purpose of Radio Group Orientation?

### DISCUSSION THREAD

**Discussion by Alek99** [radix-ui/primitives#2642] [Help] [answered]
Hello, based on the themes and primitive docs this prop is never used I would assume it would be to orient that group horizontally or vertically but in the docs, the items are just put in a flex? 

https://www.radix-ui.com/themes/docs/components/radio-group

As seen below shouldn't it just have the root as orientation='vertical'? 
<img width="837" alt="Screenshot 2024-01-08 at 2 15 35 PM" src="https://github.com/radix-ui/primitives/assets/38776361/396a7932-7529-4808-8803-61efdeda8680">

**Comment by jjenzz** [comment, marked as answer]
all props in radix are for functionality or a11y purposes only and never for controlling the styling/layout of a component. it is up to you to style the component vertically or horizontally based on the orientation you pass. the `orientation` prop simply aids screen readers.

**Reply by benoitgrelard** [reply]
@Alek99 specifically in this case, it helps setting up the right arrow keys for keyboard navigation.
